x/curating/keeper/upvote.go

```diff
@@ -131,3 +131,22 @@ func (k Keeper) IterateUpvotes(
 		}
 	}
 }
+
+// IterateUpvotes performs a callback function for each upvote
+func (k Keeper) IterateAllUpvotes(
+	ctx sdk.Context, vendorID uint32, cb func(upvote types.Upvote) (stop bool)) {
+
+	store := ctx.KVStore(k.storeKey)
+
+	// iterator over upvoters on a post
+	it := sdk.KVStorePrefixIterator(store, nil)
```

Comment:
@shanev, I ran into the same problem again trying to get all upvotes.

I thinkg the problem is `sdk.KVStorePrefixIterator(store, nil)` in both instances. How do I iterate over everything in a store? Maybe I should use something else beside `KVStorePrefixIterator`?